### DISCUSSION TITLE
タグ候補表示に 名寄せされたタグ も表示して欲しい

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#68] [要望・アイデア・不具合]
タグに RubyOnRails を指定しても Rails に名寄せ（自動変更）されます。
タグ入力時、RubyOn まで入力して手を止めると、

* RubyOnRails5.2
* RubyOnRails6.0
* RubyOnRalis （←じつはスペルミスしているので指定して欲しくない）

などが候補表示され、RubyOnRails が表示されません。
そのため、RubyOnRails5.2 や スペルミスしている RubyOnRalis を指定する記事が後を絶ちません。
そこで、名寄せ登録されている RubyOnRails を先頭に表示するようにしていただけないでしょうか。

**Comment by kskwtnk** [comment]
@shiracamus 

ご意見ありがとうございます！
影響の調査や誤字のあるタグの名寄せ設定で少し時間がかかりそうですが、対応いたします！

**Reply by kskwtnk** [reply]
https://github.com/increments/qiita-discussions/issues/70

Issueを作成いたしました 💪

**Comment by kyntk** [comment]
@shiracamus 
意見をいただいた内容ですが、以下のissueで対応しリリースしました!
ご意見いただきありがとうございました!
https://github.com/increments/qiita-discussions/issues/70

**Reply by shiracamus** [reply]
対応ありがとうございました。

**Reply by shiracamus** [reply]
んーーー
rubyon まで入力したら、Rails だけじゃなく、Rails5 と Rails6 も候補に出るようにならないでしょうか？

**Reply by kskwtnk** [reply]
こちらについては今のところ対応の予定はございません 🙇 

我々としてはバージョンが具体的なタグをつけるときは`Rails` `Rails6`といった具合で2つつけていただくのが検索性などを考えると良いのかなとと思っています
（ただしタグ数の上限が5つなので障壁になってしまうかもしれませんが……そちらも解消したいと考えています）
